ocaml/xapi/xapi_gpumon.mli

```diff
@@ -15,3 +15,12 @@
 (** Stop gpumon if it's running, perform f, then start gpumon if
   * no other threads which require gpumon to be stopped are running. *)
 val with_gpumon_stopped : ?timeout:float -> (unit -> 'a) -> 'a
+
+module Nvidia : sig
+  val key: string
+  val get_pgpu_compatibility_metadata:
+    __context:Context.t ->
+    pgpu:[ `PGPU ] API.Ref.t ->
+    (string * string Gpumon_client.Client.RPCM.t) list
```

Comment:
What is this `Gpumon_client.Client.RPCM.t`? I expected just a `(string * string) list` return type here.